cmd/podman/ps.go

```diff
@@ -458,15 +474,14 @@ func getTemplateOutput(containers []*libpod.Container, opts psOptions) ([]psTemp
 			logrus.Errorf("couldn't get some ps information, error getting artifact %q: %v", ctr.ID(), err)
 		}
 
+		ns = getNamespaces(pid)
```

Comment:
Does this blow up if pid == 0 because we never got it? Maybe put in if opts.namespace?